app/src/main/java/ai/elimu/appstore/synchronization/AppListAdapter.java

```diff
@@ -181,6 +182,18 @@ public void onClick(View view) {
                             ".elimu-ai/appstore/apks/" + language);
 
                     final File apkFile = new File(apkDirectory, fileName);
+
+                    /**
+                     * Get all local versions of current APK file for deleting
+                     */
+                    final File[] apkFiles = apkDirectory.listFiles(new FilenameFilter() {
+                        @Override
+                        public boolean accept(File file, String name) {
+                            return (name.startsWith(applicationVersion.getApplication()
+                                    .getPackageName())
+                                    && name.endsWith(".apk"));
+                        }
+                    });
                     Timber.i("apkFile: " + apkFile);
```

Comment:
Move this line above the newly added code block. Or move the `apkFile` variable below.